src/VisualStudio/VisualBasic/Impl/Debugging/DataTipInfoGetter.vb

```diff
@@ -38,33 +38,32 @@ Namespace Microsoft.VisualStudio.LanguageServices.VisualBasic.Debugging
                     Return Nothing
                 End If
 
+                Dim conditionalAccess As ExpressionSyntax = Nothing
                 If expression.IsRightSideOfDotOrBang() Then
-                    Dim parent = DirectCast(expression.Parent, ExpressionSyntax)
-                    Dim curr = parent
+                    expression = DirectCast(expression.Parent, ExpressionSyntax)
+                    Dim curr = expression
```

Comment:
Consider inserting a blank line before the declaration of ```curr```.  I think that would make it clearer that two separate computations happen within the ```if```: one to update ```expression``` and another to update ```conditionalAccess```.